pyEcoHAB/incohort_sociability.py

```diff
@@ -353,21 +355,21 @@ def get_incohort_sociability(ecohab_data, timeline, binsize, res_dir="",
                               res_dir,
                               out_dir_rasters_add,
                               out_name % ("measured", "csv"),
```

Comment:
In lines 355-375 prefix has been removed from function arguments. Please put it back.